src/miner.cpp

```diff
@@ -364,6 +364,7 @@ bool BlockAssembler::TestPackageSigOps(uint64_t packageSize, unsigned int packag
         maxSigOpsAllowed = GetMaxBlockSigOpsCount(nBlockSize + packageSize);
     }
 
+    // Note that the may2020 rule should be > so this assembles a block with 1 less sigcheck than possible
     if (nBlockSigOps + packageSigOps >= maxSigOpsAllowed)
         return false;
```

Comment:
why not use a may2020 if/else switch here so that after the fork we're using the correct  >  rather than the >= ... this way when we go back after the fork to cleanup and take out the old may2020 switches we'll find this one and can remove the old code as well.

Reply:
out of an excess of caution.  Its safer to not go right up to the limit in case implementations mess up the <= vs < in consensus code.  Also, BCHN and ABC are doing it this way so I got cold feet. :-)